nodejs/koa/ibmiAuth.js

```diff
@@ -0,0 +1,86 @@
+const { Connection, ProgramCall } = require('itoolkit');
+const { parseString } = require('xml2js');
+
+//consider converting to promise based.
+async function QSYSGETPH(username, password) {
```

Comment:
I've always disliked the QSYSGETPH technique for this kind of task, for no reason other than "eew! icky!"

If we're using `odbc` anyway, we could just use `odbc` as the authenticator mechanism, instead of using the `*LOCAL` datasource name. Kinda like what was done here:
https://github.com/IBM/ibmi-oss-examples/blob/e22395fb880c98bbc40950727a992a87fe2660fb/nodejs/express/books/utils.js#L18

It seems a good idea to have an npm authenticator module available so that users don't need this type of code repeated everywhere.

Reply:
Changed to using ODBC as the authenticator